src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -2129,75 +2144,6 @@ private static string GetStringForPSCommand(PSCommand psCommand)
             return stringBuilder.ToString();
         }
 
-        private void SetExecutionPolicy()
```

Comment:
Huge fan of removing this and relying on the initialSessionState. I look forward to bringing the entire InitialSessionState in instead of just the LanguageMode so we more accurately create runspaces similar to the parent.